deploy.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Deploy WP site
+  hosts: web
+  remote_user: deploy
```

Comment:
I wonder if you actually want this:
`remote_user: "{{ deploy_user }}"`
but I may be getting ahead of myself.